mate-panel/menu.c

```diff
@@ -1124,34 +1127,29 @@ static void
 create_menuitem_from_alias (GtkWidget      *menu,
 			    MateMenuTreeAlias *alias)
 {
-	MateMenuTreeItem *aliased_item;
-
-	aliased_item = matemenu_tree_alias_get_item (alias);
-
-	switch (matemenu_tree_item_get_type (aliased_item)) {
+	switch (matemenu_tree_alias_get_aliased_item_type (alias)) {
 	case MATEMENU_TREE_ITEM_DIRECTORY:
 		create_submenu (menu,
-				MATEMENU_TREE_DIRECTORY (aliased_item),
+				matemenu_tree_alias_get_directory (alias),
```

Comment:
Same as in previous comment about `matemenu_tree_alias_get_directory`

Reply:
Though it was already used below in the original code... so either it shouldn't be unref'ed here, or we already had a memory leak before this PR.